Scripts/DCS-BIOS/lib/common/JSONHelper.lua

```diff
@@ -0,0 +1,47 @@
+module("JSONHelper", package.seeall)
+
+--- @class JSONHelper
+local JSONHelper = {}
+
+--- @func Encodes to JSON and writes to file
+--- @param value any
+--- @param filename string File name including path
+function JSONHelper.encode_to_file(value, filename)
+	local JSON = BIOS.json
```

Comment:
The only place BIOS.json will be used is in this file, right? I'm thinking at the very least we should get rid of BIOS.json and just keep that definition local to this file.